spring-cloud-task-core/pom.xml

```diff
@@ -60,6 +60,16 @@
 			<groupId>org.springframework.data</groupId>
 			<artifactId>spring-data-commons</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+			<version>5.0.0.RELEASE</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-jdbc</artifactId>
+			<version>5.0.0.RELEASE</version>
+		</dependency>
```

Comment:
Shouldn't these be optional dependencies?  We don't want to pull in SI if they aren't using this feature...

Reply:
If we set them as optional we get class not found exception. Are we saying that the user has to explicitly add the SI dependency if they want to use this feature?

Reply:
Well we can’t just bring in SI for applications that aren’t going to use it.  I’d expect a user that was going to use this functionality to use the `spring-cloud-starter-task` and the `spring-boot-starter-integration`.  We’d need to document that it’s needed, but if an app isn’t going to use SI, we can’t just require the dependency